src/components/AddBook.js

```diff
@@ -1,19 +1,56 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
 import styles from './AddBook.module.css';
+import { addBook } from '../redux/books/books';
 
 function AddBook() {
+  const [book, setBook] = useState({
+    title: '',
+    author: '',
+    id: `${Date.now()}`,
+  });
+  const dispatch = useDispatch();
+
+  const handleClick = (e) => {
+    e.preventDefault();
+    dispatch(addBook(book));
+  };
```

Comment:
- [ ] Hi. Currently, when trying to add a new book, the book state does not change and it adds a book without a name or title and the id remains the same please kindly fix it
<img width="520" alt="image" src="https://user-images.githubusercontent.com/37901469/200949896-99605056-07e0-4792-8b58-608c3030cfef.png">

Reply:
Resolved

Reply:
well resolved! ✔️